change/@ni-nimble-components-8499a62b-64ea-41e0-877c-6bacca7ae0c8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update TableCellView tabbableChildren to be an observable property",
```

Comment:
Closing the draft PR as the discussion is stale. FYI @msmithNI @fredvisser I'm betting this refactor is needed as part of the editable table cells implementation where I suspect new tabbable elements will be created dynamically
https://github.com/ni/nimble/pull/2477